tools/js-sdk-release-tools/src/hlc/generateMgmt.ts

```diff
@@ -60,7 +60,7 @@ export async function generateMgmt(options: {
     const changedPackageDirectories: Set<string> = await getChangedPackageDirectory(!options.skipGeneration);
     for (const changedPackageDirectory of changedPackageDirectories) {
         const packagePath: string = path.join(options.sdkRepo, changedPackageDirectory);
-        let outputPackageInfo = getOutputPackageInfo(options.runningEnvironment, options.readmeMd);
+        let outputPackageInfo = getOutputPackageInfo(options.runningEnvironment, options.readmeMd, undefined);
```

Comment:
will `getOutputPackageInfo(options.runningEnvironment, options.readmeMd)` have the same effect as `getOutputPackageInfo(options.runningEnvironment, options.readmeMd, undefined)` ?

Reply:
No, I have changed the interface of function `getOutputPackageInfo`, and the last parameter is required.